content/en/training/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Training
+menu: { main: { weight: 45 } }
+description:
+  Training programs, certifications, and support resources for OpenTelemetry
+cSpell:ignore: otca
+---
+
+{{% blocks/section color="white" %}}
+
+## {{% param title %}}
+
+This page showcases training and support resources for the OpenTelemetry
+project. Check back often for updates!
+
+### Certifications
+
+Demonstrate your expertise in OpenTelemetry by becoming an OpenTelemetry
+Certified Associate.
+
+<a href="https://www.cncf.io/training/certification/otca/">
+  <img src="lft-badge-opentelemetry-associate2.svg" style="width: 250px; height: auto;" alt="OTCA Badge">
+</a>
```

Comment:
Use markdown syntax here (so that the render hook can insert the appropriate target=blank etc).
Or I don't mind make a pass over the file.

Reply:
> so that the render hook can insert the appropriate target=blank etc

I don't know what...any...of that means. 😆 

Also, I thought you couldn't add a link to an image using Markdown?

All that's to say, yes, please go ahead and fix it however you see fit. I'll study the changes you make so I can learn.

Reply:
😄 ... ok. Let's merge this as is. I might want to propose some styling changes anyhow (the current version of this page that is like the Status page format is quite ok for now).